ostd/src/mm/page/mod.rs

```diff
@@ -189,7 +193,14 @@ impl<M: PageMeta> Page<M> {
 
 impl<M: PageMeta> Clone for Page<M> {
     fn clone(&self) -> Self {
-        self.slot().ref_count.fetch_add(1, Ordering::Relaxed);
+        let old = self.slot().ref_count.fetch_add(1, Ordering::Relaxed);
+        if old >= REF_COUNT_MAX {
```

Comment:
This pattern along with the associated comments has been repeated for three times. Let's extract them into a function or method.

Reply:
Won't be a problem since #1724 intend to remove `DynPage`.

Reply:
It seems to be appreciate to be an associated method of `MetaSlot`, though.

> Won't be a problem since #1724 intend to remove `DynPage`.

`inc_page_ref_count` is still there, right?

Reply:
Yep. Some methods indeed are associated with `MetaSlot` but scattered across the module.